usaspending_api/awards/models.py

```diff
@@ -455,7 +455,7 @@ class TransactionAssistance(DataSourceTrackedModel):
     uri = models.TextField(blank=True, null=True)
     cfda_number = models.TextField(blank=True, null=True, verbose_name="CFDA Number")
```

Comment:
Can we remove `cfda_number` and `cfda_title` fields now that cfda is properly foreign-keyed? That would also be a forcing function to make sure that API users are grabbing this info from the one, canonical place. Not sure if this would require further updates to the data loaders...didn't check on that.